### ISSUE TITLE
Idle Dim Settings not respected/saved

### ISSUE THREAD

**Issue by marc-gist** [HASwitchPlate/openHASP-custom-component#103]
## Configuration

```yaml
N/A

```

## Describe the bug

When you add a plate to HA, de-select the dim when idle setting, add the device to your configuration.yaml file, restart HA.  Everything comes up fine, however, the panel still dims on idle, and turns off on long idle!  Checking the setting in HA, the checkbox for dim is selected.  Uncheck, click save, HA reports setting saved, but when you check the box is selected again and the panel still dims on idle.

unfortunately makes this integration unusable :(

Dim on idle should be handled by automations! so a sensor for idle should be added and used ideally. or just remove this "feature" for now.

Thanks!

## Debug log

```
N/A

```

**Comment by fvanroie** [comment]
It seems Config Flow is only for initial setup, to allow the configuration to be changed later Options Flow needs to be implemented:
https://community.home-assistant.io/t/configflowhandler-and-optionsflowhandler-managing-the-same-parameter/365582

**Comment by htvekov** [comment]
Yes, we need to be able to disable the different idle events individually or just globally.

I'm fiddling with a custom code proximity sensor implementation and needed a solution for this.
So I've made a crude openHASP CC `light.py` hack to identity a single openHASP nodename and exclude `idle off` events specifically for this single device only.

![image](https://user-images.githubusercontent.com/45469813/233069189-4c51ce12-0e37-46dd-a3b1-d2f96f82ecad.png)
